wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -829,6 +833,8 @@ public void deleteProject(
       settings.put("cheWorkspaceDevfileRegistryUrl", devfileRegistryUrl);
     }
 
+    settings.put(CHE_WORKSPACE_PERSIST_VOLUMES_PROPERTY, Boolean.toString(defaultPersistVolumes));
```

Comment:
Previously, a dedicated setting property was used for propagating this value but I reverted it https://github.com/eclipse/che/pull/15775/commits/0c81158738365dd6226a691370d08eaeaadec714#diff-3cc9e73374c2afcf0ddf10962bee795bL52
Why? Personally, I can only one prop of using dedicated property - we are able to rename property on Che Server side without affecting REST client.
On another hand, with reusing the same property - users are able to find easily more information about this settings and how it can be configured on the Che Server side.